examples/speculative_decoding/main.py

```diff
@@ -62,11 +71,29 @@ class DataArguments:
         metadata={"help": "Path to the training data."},
     )
     eval_data_path: str = field(default=None, metadata={"help": "Path to the evaluation data."})
+    offline_data_path: str = field(
+        default=None,
+        metadata={"help": "Path to the offline training data. See `offline_training`."},
+    )
     lazy_preprocess: bool = True
     draft_vocab_cache_dir: str = field(
         default="draft_vocab_cache",
         metadata={"help": "Path to the d2t cache directory."},
     )
+    offline_training: bool = field(
```

Comment:
It seems to me that we can use this [existing argument from EagleConfig](https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/b913290dfd21ead6577dae7a0cdbe4ffc24a6729/modelopt/torch/speculative/config.py#L70) to switch between online/offline training.